### ISSUE TITLE
Route prefix is duplicating

### ISSUE THREAD

**Issue by wodka** [symfony/symfony#25746]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             |  no
| Symfony version  | 4.0.3

If loading multiple templates via glob with a prefix the prefix is beeing duplicated because RouteCollectionBuilder::build is called multiple times.

Example:

```yml
# config/routes/other/a.yaml
controller_a:
    resource: ../../../src/Controller/AController
    type: annotation
```
```yml
# config/routes/other/b.yaml
controller_b:
    resource: ../../../src/Controller/BController
    type: annotation
```

and then in the Kernel: 
```php
protected function configureRoutes(RouteCollectionBuilder $routes){
    $confDir = $this->getProjectDir().'/config';
    /* other routes removed */
    $routes->import($confDir.'/routes/other/**/*'.self::CONFIG_EXTS, '/other/', 'glob');
}
```

will create the routes:
```
controller_a /other/other/a
controller_b /other/other/b
```

by doing this the duplication can be reproduced:
```php
protected function configureRoutes(RouteCollectionBuilder $routes){
    $confDir = $this->getProjectDir().'/config';
    /* other routes removed */
    $b = $routes->import($confDir.'/routes/other/**/*'.self::CONFIG_EXTS, '/other/', 'glob');
    $b->build();
}
```

will create the routes:
```
controller_a /other/other/other/a
controller_b /other/other/other/b
```

**Comment by wodka** [comment]
it's important to have multiple files matching for glob -> each new file will add another duplicate to the prefix

**Comment by sroze** [comment]
So you have the duplication when calling `build`, correct?

**Comment by wodka** [comment]
I'm not calling build myself - figured out that by manually calling build I can further duplicate the prefix

**Comment by sroze** [comment]
I can reproduce the bug you've mentioned.

Status: Reviewed

**Comment by sroze** [comment]
@wodka I've managed to fix it on my side with this: #25791. Can you confirm the following fix works for you as well?

**Comment by wodka** [comment]
works for me
